### PR TITLE
New: Initial OneZero

### PR DESCRIPTION
**Pull request by shaiu**

### CODE REVIEW

**Review by baruchiro — APPROVED** [review]
Approved by please see my comments and response them before continue.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  Put an `x` in the boxes that apply. You can also fill these out after
+  creating the PR. If you're unsure about any of them, don't hesitate to ask.
+  We're here to help! This is simply a reminder of what we are going to look
+  for before merging your code.
+-->
+
+- [ ] The code change is tested and works locally.
+- [ ] Local tests pass. **Your PR cannot be merged unless tests pass**
+- [ ] There is no commented out code in this PR.
+- [ ] The code has been formatted
```
[the remark this answers is not in the document]
Comment (shaiu) [reply .github/PULL_REQUEST_TEMPLATE.md]: Removed. I'll do this in a different PR

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
+<!--
```
[the remark this answers is not in the document]
Comment (shaiu) [reply .github/PULL_REQUEST_TEMPLATE.md]: Removed. I'll do this in a different PR

File: package.json [hunk 3]
```diff
@@ -22,6 +22,7 @@
     "test:preload": "vitest run -r packages/preload --passWithNoTests",
     "test:renderer": "vitest run -r packages/renderer --passWithNoTests",
     "watch": "node scripts/watch.js",
+    "watch:debug": "node scripts/watch.js --inspect-brk",
```
[the remark this answers is not in the document]
Comment (shaiu) [reply package.json]: Removed

File: packages/renderer/src/accountMetadata.tsx [hunk 4]
```diff
@@ -102,6 +108,8 @@ export const LOGIN_FIELD_MIN_LENGTH = {
   [NUM_FIELD]: 4,
   [CARD_SIX_DIGITS_FIELD]: 6,
   [NATIONAL_ID_FIELD]: 9,
+  [EMAIL_FIELD]: 3,
+  [PHONE_FIELD]: 12,
```
[the remark this answers is not in the document]
Comment (shaiu) [reply packages/renderer/src/accountMetadata.tsx]: Done

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (baruchiro) [reply .github/PULL_REQUEST_TEMPLATE.md]: Oh I marked only the `The code has been formatted` row, not everything 😂

But actually Github is right, the commented-out and tests can be auto-tested too

**Pull request merged**